README.md

```diff
@@ -91,20 +84,34 @@ For macOS Catalina, open the configuration of zsh via the terminal:
 * Type `nano ~/.zshrc` to open the configuration of the terminal
 * Add the path to your mono installation: `export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/lib/pkgconfig:/Library/Frameworks/Mono.framework/Versions/6.12.0/lib/pkgconfig:$PKG_CONFIG_PATH`. Make sure that the Path matches to your version (Here 6.12.0)
 * Save everything and execute `. ~/.zshrc` 
-4. Navigate to the alphapept folder and install the package with `pip install .` (default users) or with `pip install -e .` to enable developers mode.
+4. Install AlphaPept via pip: `pip install "alphapept[gui-stable]"`. If you want to use AlphaPept as a package withouth the GUI dependencies and strict version dependencies use `pip install alphapept`.
 
+#### Developer
+1. Redirect to the folder of choice and clone the repository: `git clone https://github.com/MannLabs/alphapept.git`
+2. Navigate to the alphapept folder with `cd alphapept` and install the package with `pip install .` (default users) or with `pip install -e .` to enable developers mode. Note that you can use the different requirements here aswell (e.g. `pip install ".[gui-stable]"`)
 
-### Standalone Windows Installer
-To use AlphaPept as a stand-alone program for end-users, it can be installed on Windows machines via a one-click installer. Download the latest version [here](https://github.com/MannLabs/alphapept/releases/latest).
+#### GPU Support
+To enable the usage of a GPU, additional packages need to be installed. The following instructions are targeted at a more experienced audience.
```

Comment:
"Experienced" should be renamed, not to offend people or create expectations (it is fairly subjective)